apps/site/src/app/(main)/(home)/sections/ChooseCharacter/ChooseCharacter.tsx

```diff
@@ -14,7 +14,7 @@ import styles from "./CharacterBox.module.scss";
 
 const ChooseCharacter = () => {
 	return (
-		<section className="flex flex-col justify-center my-14">
+		<section className="flex flex-col justify-center my-14" id="apply">
```

Comment:
Note for later: didn't think you would add it now, but I think this can be used now instead of what #437 did.